acrobat/blocks/app-banner/app-banner.js

```diff
@@ -0,0 +1,128 @@
+import { createTag } from "../../scripts/miloUtils.js";
+
+var link = '';
+var mobileOS = getMobileOperatingSystem();
+
+
+export default async function init(el) {
+  //Exit if not mobile/tablet 
+  if (mobileOS === 'unknown') {
+    el.innerHTML = '';
+    return;
+  }
+
+  createTag.then((createTag) => {
+    const bannerPlaceHolder = createTag('div', { class: 'placeholder-banner' }, '');
```

Comment:
` createTag(tag, attributes, html)` doesn't require `attributes` or `html`. You can remove the empty string.
If this is repeated please do the same. 
```
const bannerPlaceHolder = createTag('div', { class: 'placeholder-banner' });
```

Reply:
Done.